Compass129Sources/COMfile/C12COM9.ASC

```diff
@@ -876,38 +876,38 @@ m_free_it	inc	hl
 
 m_alloc_notpa	ld	a,(stat_mem)
 	cp	2
-	jr	nc,m_alloc_mntpa	;iets moeilijker bij memman
-	ld	c,#10	;dos2:zeker geen tpa
+	jr	nc,m_alloc_mntpa	;a little more difficult with memman
+	ld	c,#10	;dos2:definitely no tpa
 	jr	m_all_save	;=call m_all_save, ret
 m_alloc_mntpa	ld	b,0
 m_try_nottpa	push	bc
 	ld	b,#03	;memman:fseg
 	call	m_all_save
 	jr	c,m_alles_op
-	ld	a,(#f344)	;controle op memman: was het wel een
-	cp	b	;niet-tpa?
+	ld	a,(#f344)	;check on memman: was it a
+	cp	b	;non-tpa?
 	jr	nz,m_ja_eindelyk
 	pop	bc
 	djnz	m_try_nottpa
 	di
-	halt		;hier wordt er normaal nooit gekomen, bij memman kunnen
-	;er nooit 256 tpasegm worden aangevraagd aangezien een mapper
-	;maar 256 segm telt en 0 nooit vrijgegeven wordt.
-m_ja_eindelyk	pop	bc	;haal b
+	halt		;Normally we'll never reach this, could with memman
+	;256 tpasegm are never requested since a mapper
+	;counts only 256 segments and 0 is never freed.

```

Comment:
"Normally we'll never reach this point, as you can never request 256 tpasegm with memman, since a mapper only counts 256 segm, and 0 is never freed".